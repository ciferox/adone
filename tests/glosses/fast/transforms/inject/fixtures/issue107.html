<!DOCTYPE html>
<html>
<head>
  <title>gulp-inject</title>
</head>
<body>

  <!-- inject:js-->
  <!-- endinject-->
</body>
</html>