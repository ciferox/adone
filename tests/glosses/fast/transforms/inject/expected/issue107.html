<!DOCTYPE html>
<html>
<head>
  <title>gulp-inject</title>
</head>
<body>

  <!-- inject:js-->
  <script src="/fixtures/lib.js?v=1.0.0"></script>
  <!-- endinject-->
</body>
</html>